app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::ItemsController < Api::ApiController
+  respond_to :json
+
+  before_action :authenticate
```

Comment:
Move `before_action :authenticate` to parent class `Api::ApiController` so all api-related controller actions will require authentication by default. This is a more secure default, it is less likely to result in new API controllers that accidentally don't have authentication. Similar line in the `ListsController`.